rfhub2/cli/api_client.py

```diff
@@ -0,0 +1,63 @@
+from requests import session, post, get, delete
+from typing import Dict
+
+
+API_V1 = 'api/v1'
+TEST_COLLECTION = {"name": "healtcheck_collection", "type": "a", "version": "a", "scope": "a",
+                   "named_args": "a", "path": "a", "doc": "a", "doc_format": "a"}
+
+
+class Client(object):
+    """
+    API client with methods to populate rfhub2 application.
+    """
+    def __init__(self, app_url: str, user: str, password: str):
+        self.app_url = app_url
+        self.session = session()
+        self.api_url = f'{self.app_url}/{API_V1}'
+        self.session.auth = (user, password)
+        self.session.headers = {"Content-Type": "application/json", "accept": "application/json"}
+
+    def get_collections(self) -> Dict:
+        """
+        Gets list of collections object using request get method.
+        """
+        return self._get_request(endpoint='collections')
+
+    def add_collection(self, data: Dict) -> Dict:
+        """
+        Adds collection using request post method.
+        """
+        return self._post_request(endpoint='collections', data=data)
+
+    def delete_collection(self, id: int) -> delete:
```

Comment:
type annotation of return value is Response from requests

Reply:
fixed